### PR TITLE
chore(CVR): enable REBUILD_ESTIMATES feature gate

### PR DESCRIPTION
**Pull request by mittachaitu**
Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

**What this PR does / why we need it**:
- Enables the feature gate REBUILD_ESTIMATES by default(previous PRs https://github.com/openebs/maya/pull/1639, https://github.com/openebs/maya/pull/1641). 
- Removes extra fields under CVR.Status.Snapshots, current it has only LogicalReferenced.
**LogicalReferenced**: Size in bytes of the volume when the snapshot was taken.

**Example CVR YAML**:
```yaml
apiVersion: openebs.io/v1alpha1
kind: CStorVolumeReplica
metadata:
  annotations:
    cstorpool.openebs.io/hostname: 127.0.0.1
    isRestoreVol: "false"
    openebs.io/storage-class-ref: |
      name: single-sc
      resourceVersion: 20797
  creationTimestamp: "2020-04-15T18:20:34Z"
  finalizers:
  - cstorvolumereplica.openebs.io/finalizer
  generation: 27
  labels:
    cstorpool.openebs.io/name: cstor-sparse-pool-g45t
    cstorpool.openebs.io/uid: fab7b50f-29ea-495b-bfc4-cd256ccb1d27
    cstorvolume.openebs.io/name: pvc-15ec16c1-49a6-43be-ad26-e1ce4229b7e7
    openebs.io/cas-template-name: cstor-volume-create-default-1.9.0
    openebs.io/persistent-volume: pvc-15ec16c1-49a6-43be-ad26-e1ce4229b7e7
    openebs.io/version: 1.9.0
  name: pvc-15ec16c1-49a6-43be-ad26-e1ce4229b7e7-cstor-sparse-pool-g45t
  namespace: openebs
  resourceVersion: "22857"
  selfLink: /apis/openebs.io/v1alpha1/namespaces/openebs/cstorvolumereplicas/pvc-15ec16c1-49a6-43be-ad26-e1ce4229b7e7-cstor-sparse-pool-g45t
  uid: 03f5be25-4ada-4dba-bb79-b5858f99e8d7
spec:
  capacity: 5G
  replicaid: 1733F78111FE8F998284A7DA13998CF5
  targetIP: 10.0.0.133
  zvolWorkers: ""
status:
  capacity:
    totalAllocated: 6K
    used: 6K
  lastTransitionTime: "2020-04-15T18:29:23Z"
  lastUpdateTime: "2020-04-15T18:38:48Z"
  phase: Healthy
  pendingSnapshots:
    vol1_snap3:
      logicalReferenced: 417277952
  snapshots:
    vol1_snap2:
      logicalReferenced: 6144
versionDetails:
  autoUpgrade: false
  desired: 1.9.0
  status:
    current: 1.9.0
    dependentsUpgraded: true
    lastUpdateTime: null
    state: ""
```

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
- As of now used value under the snapshot will not increase(need to fix it). 

**Release Note**:
- Enables `REBUILD_ESTIMATES` feature gate by default so that CVR status will have a list of snapshots and pending snapshots. The list under snapshots represents snapshots created for that volume and pendingSnapshots represents snapshots that are yet to rebuild from the peer replicas.

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: cmd/cstor-pool-mgmt/volumereplica/volumereplica.go [hunk 1]
```diff
@@ -718,7 +718,7 @@ func GetAndUpdateSnapshotInfo(
 	//   as is to cover this corner case below check is required.
 	if cvr.Status.Phase == apis.CVRStatusOnline &&
 		len(cvr.Status.PendingSnapshots) != 0 {
-		klog.Infof("CVR: %s is marked as %s hence removing pending snapshots %v",
+		klog.V(2).Infof("CVR: %s is marked as %s hence removing pending snapshots %v",
```
Comment (vishnuitta) [inline cmd/cstor-pool-mgmt/volumereplica/volumereplica.go:721]: no need to change loglevel as these are once in a while logs

File: cmd/cstor-pool-mgmt/controller/replica-controller/handler.go [hunk 2]
```diff
@@ -648,12 +648,11 @@ func (c *CStorVolumeReplicaController) syncCVRStatus(cvr *apis.CStorVolumeReplic
...
 	}
 
-	if os.Getenv(string(common.RebuildEstimates)) == "true" {
-		err = volumereplica.GetAndUpdateSnapshotInfo(c.clientset, cvr)
-		if err != nil {
-			return errors.Wrapf(err, "Unable to update snapshot list details in CVR")
-		}
+	err = volumereplica.GetAndUpdateSnapshotInfo(c.clientset, cvr)
+	if err != nil {
+		return errors.Wrapf(err, "Unable to update snapshot list details in CVR")
```
Comment (mynktl) [inline cmd/cstor-pool-mgmt/controller/replica-controller/handler.go:653]:
```suggestion
		return errors.Wrapf(err, "unable to update snapshot list details in CVR")
```

File: cmd/cstor-pool-mgmt/volumereplica/volumereplica.go (hunk 1 above)
Comment (mittachaitu) [reply cmd/cstor-pool-mgmt/volumereplica/volumereplica.go]: Done

File: cmd/cstor-pool-mgmt/controller/replica-controller/handler.go (hunk 2 above)
Comment (mittachaitu) [reply cmd/cstor-pool-mgmt/controller/replica-controller/handler.go]: Updated
